banyand/metadata/schema/etcd.go

```diff
@@ -83,6 +83,7 @@ func (eh *eventHandler) InterestOf(kind Kind) bool {
 
 type etcdSchemaRegistry struct {
 	server   *embed.Etcd
+	client   *clientv3.Client
```

Comment:
Because etcd client init at 
https://github.com/apache/skywalking-banyandb/blob/c1302224dcb7e4113b0aef59d9005dd38f1eee30/banyand/metadata/schema/etcd.go#L168
I need to convert the temporary variable to a property of `etcdSchemaRegistry `so that I can close the client resource.